test/acceptance/test/add_tests.go

```diff
@@ -1000,7 +1113,37 @@ var _ = Describe("Weave GitOps Add Tests", func() {
 			Expect(folderOutput).Should(ContainSubstring("apps"))
 			Expect(folderOutput).Should(ContainSubstring("targets"))
 		})
+
+		By("When I check for apps list", func() {
+			listOutput, _ = runCommandAndReturnStringOutput(WEGO_BIN_PATH + " app list")
+		})
+
+		By("Then I should see appNames for both apps listed", func() {
+			Eventually(listOutput).Should(ContainSubstring(appName1))
+			Eventually(listOutput).Should(ContainSubstring(appName2))
+		})
+
+		By("When I check the app status for "+appName1, func() {
+			appStatus1 = runCommandAndReturnSessionOutput(fmt.Sprintf("%s app status %s", WEGO_BIN_PATH, appName1))
+		})
+
+		By("Then I should see the status for "+appName1, func() {
+			Eventually(appStatus1).Should(gbytes.Say(`Last successful reconciliation:`))
+			Eventually(appStatus1).Should(gbytes.Say(`helmrepository/` + appName1))
+			Eventually(appStatus1).Should(gbytes.Say(`helmrelease/` + appName1))
```

Comment:
Suggestion: Make these checks flexible by using `ContainsSubstring` instead of gbytes.Say